sshd/zenith/sshd/tunnel.py

```
@@ -373,36 +380,33 @@ def consul_heartbeat(
         elif liveness_failures > 0:
```

surely we should we do a warning message if we move the status to critical?

---

Maybe if we do that we should raise an exception after setting the service to critical to trigger a reconnect??

---

I guess we get a warning log now on line 385, cool.